### DISCUSSION TITLE
Chrome complains about website not being secure when using URL like something.com.ddev.site

### DISCUSSION THREAD

**Discussion by BernhardBaumrock** [ddev/ddev#5051] [General]
First of all thank you for creating this awesome product!

I'm using hostnames like domain.com.ddev.site for my local development. This has been working all the time until recently chrome showed me a warning that "someone might be tricking me" or the like.

I understand why chrome is doing that and I'd be fine with clicking away that warning, but on the live domain chrome says it is insecure as well:

<img width="347" alt="image" src="https://github.com/ddev/ddev/assets/8488586/31b57129-709a-461c-80cc-d0dd2c232d08">

The thing is that the certificate is valid and other browsers don't complain, so I guess chrome somehow flags that domain and marks it as insecure.

Has anyone else experienced that?

**Comment by rfay** [comment]
Can you demonstrate this when not using `<domain.com>.ddev.site`? For example, if you just use `domain.ddev.site` is it OK? I imagine Chrome is protecting you from the many garbage sites that fool you into thinking you're at a "real" URL and in reality you're on their malware site.

**Comment by BernhardBaumrock** [comment]
Hey @rfay exactly. It's a little hard to reproduce (or at least I don't know how I could do that) and I'll have to keep an eye on it over time. Just thought I'd mention it here to see if anybody else is having issues or maybe already a solution.

**Reply by rfay** [reply]
If you click the "Learn more " link, does it give you any help? Can you please provide the "learn more" link here?

**Reply by BernhardBaumrock** [reply]
That's what I did. Unfortunately that was not helpful: https://support.google.com/chrome/answer/95617?visit_id=638237444053843333-4137688560&p=ui_security_indicator&rd=1

**Comment by rfay** [comment]
You don't get the full screen warning right? 

> Dangerous: Avoid this site. If you get a full-page red warning screen, the site has been flagged as unsafe by [Safe Browsing](https://transparencyreport.google.com/safe-browsing/overview). Using the site will likely put your private information at risk.

**Comment by BernhardBaumrock** [comment]
Just had it again! This is how the warning looks like for baumrock.com.ddev.site:

<img width="707" alt="image" src="https://github.com/ddev/ddev/assets/8488586/cc2c491c-234f-4b95-9575-43e4ab78ffe4">

baumrock.com seems to work as always though. Not sure what's going on. I'll keep an eye and report back!

**Reply by rfay** [reply]
Thanks for the followup. It's definitely Chrome picking on you about your choice of URLs :)

**Comment by graham73may** [comment]
For some reason this has recently become a problem for me too: 

<img width="756" height="393" alt="image" src="https://github.com/user-attachments/assets/c5f1e7cd-4142-49d5-a89b-ccaef23d6fc8" />

---

Gemini's advice: 

That is a common (and slightly over-protective) feature in Chrome called **Lookalike URL Navigation Suggestions**.

Since DDEV uses custom local domains (like .ddev.site), Chrome’s safety engine often mistakes them for "typosquatting" or phishing attempts against real, similar-sounding .org or .com websites.

Here is how you can disable this specific warning so you can get back to coding in peace.

**Method 1: Disable the Flag (The "Nuclear" Option)**
If you want to stop Chrome from checking for lookalike URLs entirely, you can toggle the internal browser flag.
	1.	Open Chrome and type chrome://flags in the address bar.
	2.	Search for "Lookalike URL navigation suggestions".
	3.	Set the dropdown to Disabled.
	4.	Relaunch Chrome.

**Method 2: Adjust "Safe Browsing" Settings**
If you don't want to mess with flags, you can lower the sensitivity of the protection in your standard settings:
	1.	Go to chrome://settings/security.
	2.	Look at your Safe Browsing level.
	3.	If you have "Enhanced protection" turned on, it is much more aggressive about flagging local development URLs.
	4.	Switching to "Standard protection" usually resolves the issue while still keeping you safe from known malicious sites.

**Method 3: Use the "Ignore" Button (Temporary)**
If you click the "Ignore" button on that warning page, Chrome should remember your choice for that specific local domain for the duration of your session. However, as a developer, this is usually annoying because it may reset after you clear your cache or restart the browser.

**Reply by graham73may** [reply]
Ok, turns out my Safe Browsing settings are at an organisation level, so had to do this instead: 

Since your "Safe Browsing" is locked at the organizational level, your account is being managed as part of a "managed browser" session. Here is how you (or your admin) can whitelist .ddev.site:

**1. Access the Admin Console**
If you have admin rights, go to [admin.google.com](https://admin.google.com/). If you don't, you'll need to send these steps to the person who manages your company's email/Google account.

**2. Navigate to Chrome Policies**
	1.	In the sidebar, go to Devices > Chrome > Settings.
	2.	Select Users & browsers from the menu.
	3.	If your company has different departments (Organizational Units), make sure the correct one is selected on the left.

**3. Add the Whitelist Policy**
	1.	Use the "Search settings" bar at the top and type: Lookalike.
	2.	Look for the setting: Lookalike warning allowlist.
	3.	In the text box, enter: ddev.site
	4.	Click Save in the top right corner.

**How to Force the Update on Your Machine**
Once the setting is saved in the Admin Console, it won't always apply to your Chrome browser instantly. To force it:
	1.	Open a new tab and go to chrome://policy.
	2.	Click the Reload policies button.
	3.	Search the list for LookalikeWarningAllowlistDomains.
	4.	If it appears with ddev.site in the value column, you're good to go!

<img width="1409" height="657" alt="image" src="https://github.com/user-attachments/assets/96c253da-ec3c-4bdd-82aa-02b37258d04d" />
